tests/discovery/test_url_to_graph.py

```diff
@@ -55,7 +55,7 @@ def test_download_uri_cases():
         graph = get_graph_for_format(uri, formats=[format])
         assert isinstance(graph, Graph)
         assert len(graph) > 0
-        assert len(graph) == case["length"]
+        assert len(graph) >= case["length"]
```

Comment:
motivation?
why should test be made more lenient?  
are we allowing errors/issues to go undetected through this?